### ISSUE TITLE
[0.9.1.0 beta release-105]French localization crash in contracts

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#19091]
Using french localization and posting a contract for different clauses causing server crash.
![Screenshot_14.jpg](https://images.zenhubusercontent.com/5f5a7fa59878ed957a861886/9cc5afb4-27c3-4c99-85cb-b915688cca88)

System.FormatException: Input string was not in a correct format.
   at System.Text.StringBuilder.FormatError()
   at System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.Format(String format, Object[] args)
   at Eco.Shared.Localization.Localizer.LocalizeFormattableString(FormattableString fs)
   at Eco.Shared.Localization.Localizer.ToString(LocString locString)
   at Eco.Gameplay.Economy.Contracts.ClauseListExtensions.<>c.<ClauseDesc>b__2_0(ContractClause x)
   at System.Linq.Enumerable.SelectIListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.String.Join(String separator, IEnumerable`1 values)
   at Eco.Shared.Utils.StringExtensions.NewlineList(IEnumerable`1 e)
   at Eco.Gameplay.Economy.Contracts.ClauseListExtensions.ClauseDesc(IClauseList list)
   at Eco.Gameplay.Economy.Contracts.Contract.PostJob(User user)
   at Eco.Gameplay.Economy.Contracts.Contract.PostJob(Player player)</size>

**Comment by SlayksWood** [comment]
Another server crash with contract. According to information from players, the server crashed when one player press 'post' button. Second time they can't reproduce this crash.
Server and Client Localization: Russian.
Contract was created on 9.1.0 but was posted on 9.1.1
```
--BEGIN DUMP--
Dump Time
11/04/2020 13:16:02

Exception
Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Economy.Contracts.Contract.<>cDisplayClass106_0.<Open>b0(IOpenHandler x)
   at Eco.Shared.Utils.EnumerableExtensions.ForEach[T](IEnumerable1 enumeration, Action1 action)
   at Eco.Gameplay.Economy.Contracts.Contract.Open(Player player)
   at Eco.Gameplay.Economy.ContractBoardComponent.CreateAndDisplayWorkable(User user, Workable workable)
   at Eco.Gameplay.Economy.ContractBoardComponent.AddContract(Player player)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_1(Object state)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

**Comment by WeaselDog** [comment]
@mirasrael @Quadbro I am assigning you both so whoever gets to it first.  This is crucial to fix ASAP and put directly into release

**Comment by mirasrael** [comment]
I guess original issue already fixed in 9.1.2.

**Comment by mirasrael** [comment]
Yes, checked. It now fallbacks to default English message and reports error to server log:
```
[12:43:58] Invalid localization in French for `Add blocks from the property at {0} until there are more than {1}.`: `{0} bloque la propriété à {1} jusqu'à ce qu'il y ait {: greater/less} que .`
```
